core/catalog/loaders/base.py

```
@@ -0,0 +1,30 @@
+"""
+Base catalog loader utilities (PR-7).
+
+RU: Базовые утилиты для загрузчиков каталога.
+EN: Base utilities for catalog loaders.
+"""
+
+from __future__ import annotations
+
+import csv
+from pathlib import Path
+from typing import Any
```

Flake8 (7.3.0)

[error] 'typing.Any' imported but unused

(F401)